.travis.yml

```diff
@@ -9,7 +9,7 @@ cache:
     - $HOME/.sbt
     - $HOME/.ivy2
 script:
-  - sbt "set parallelExecution in Test := false" clean coverage doc assembly
+  - sbt clean scapegoat doc coverage package
```

Comment:
`sbt clean scapegoat doc coverage test` works fine on master. Please keep in mind that `coverage` inserts instructions into the code. When you package for deployment, you should clean before: `sbt clean package`.